docs/excel/excel-add-ins-notes.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Work with notes using the Excel JavaScript API
+description: Information on using the APIs to add, remove, and edit notes.
+ms.date: 04/04/2025
+ms.localizationpriority: medium
+---
+
+# Work with notes using the Excel JavaScript API
+
+This article describes how to add, change, and remove notes in a workbook with the Excel JavaScript API. You can learn more about notes from the [Insert comments and notes in Excel](https://support.microsoft.com/office/bdcc9f5d-38e2-45b4-9a92-0b2b5c7bf6f8) article. For information about the differences between notes and comments, see [The difference between threaded comments and notes](https://support.microsoft.com/office/the-difference-between-threaded-comments-and-notes-75a51eec-4092-42ab-abf8-7669077b7be3).
+
+Notes are tied to an individual cell. Anyone viewing the workbook with sufficient permissions can view a note. Notes in a workbook are tracked by the `Workbook.notes` property. This includes notes created by users and also notes created by your add-in. The `Workbook.notes` property is a [NoteCollection](/javascript/api/excel/excel.notecollection) object that contains a collection of [Note](/javascript/api/excel/excel.note) objects. Notes are also accessible at the [Worksheet](/javascript/api/excel/excel.worksheet) level.
+
+> [!TIP]
+> To learn about adding and editing comments with the Excel JavaScript API, see [Work with comments using the Excel JavaScript API](excel-add-ins-comments.md).
+
+## Add a note
+
+Use the `NoteCollection.add` method to add notes to a workbook. This method takes two parameters:
+
+- `cellAddress`: The cell where the comment is added. This can either be a string or [Range](/javascript/api/excel/excel.range) object. The range must be a single cell.
+- `content`: The comment's content, as a string.
+
+The following code sample shows how to add a note to the selected cell in a worksheet.
+
+```js
+await Excel.run(async (context) => {
+    // This function adds a note to the selected cell.
+    const selectedRange = context.workbook.getSelectedRange();
+
+    // Note that an InvalidArgument error is thrown if multiple cells are selected.
+    context.workbook.notes.add(selectedRange, "The first note.");
+    await context.sync();
+});
+```
+
+## Change note visibility
+
+By default, the content of notes is hidden unless a user hovers over the cell with the note or sets the workbook to display notes. To programmatically display a note, use the [Note.visible](/javascript/api/excel/excel.note#excel-excel-note-visible-member) property. The following code sample shows how to change the visibility of a note.
+
+```js
+await Excel.run(async (context) => {
+    // This function sets the note on cell A1 to visible.
+    const sheet = context.workbook.worksheets.getItem("Notes");
```

Comment:
```suggestion
    const sheet = context.workbook.worksheets.getActiveWorksheet();
```
Unless a previous sample set up the "Notes" workbook I think it's simpler to just use the active one.